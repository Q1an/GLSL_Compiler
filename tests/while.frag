while ( ip < z )
  ip = ip + 1;
