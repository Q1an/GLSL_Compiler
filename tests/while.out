while        line 1 cols 1-5 is T_While 
(            line 1 cols 7-7 is '(' 
ip           line 1 cols 9-10 is T_Identifier 
<            line 1 cols 12-12 is '<' 
z            line 1 cols 14-14 is T_Identifier 
)            line 1 cols 16-16 is ')' 
ip           line 2 cols 3-4 is T_Identifier 
=            line 2 cols 6-6 is '=' 
ip           line 2 cols 8-9 is T_Identifier 
+            line 2 cols 11-11 is '+' 
1            line 2 cols 13-13 is T_IntConstant (value = 1)
;            line 2 cols 14-14 is ';' 
